apps/aering/src/aer_constants.erl

```diff
@@ -0,0 +1,10 @@
+-module(aer_constants).
```

Comment:
This could be moved into aer_parser as a function (constant_string or similar).
There should only be one call to this function as far as I know.
Can be fixed as a seprate ticket later.